projects/packages/jetpack-mu-wpcom/src/features/verbum-comments/class-verbum-comments.php

```diff
@@ -0,0 +1,594 @@
+				'must_log_in'          => '',
+				'logged_in_as'         => '',
+				'comment_notes_before' => '',
+				'comment_notes_after'  => '',
+				'title_reply'          => __( 'Leave a Reply', 'jetpack-mu-wpcom' ),
+				/* translators: % is the original posters name */
+				'title_reply_to'       => __( 'Leave a Reply to %s', 'jetpack-mu-wpcom' ),
+				'cancel_reply_link'    => __( 'Cancel reply', 'jetpack-mu-wpcom' ),
+			)
+		);
+	}
+
+	/**
+	 * Set comment reply link.
+	 * This is to fix the reply link when comment registration is required.
+	 *
+	 * @param  string $reply_link - HTML for reply link.
+	 * @param  array  $args - Default options for reply link.
+	 * @param  object $comment - Comment being replied to.
+	 * @param  object $post - PostID or WP_Post object comment is going to be displayed on.
+	 */
+	public function comment_reply_link( $reply_link, $args, $comment, $post ) {
+		// This is only necessary if comment_registration is required to post comments
+		if ( ! get_option( 'comment_registration' ) ) {
+			return $reply_link;
+		}
+
+		$comment    = get_comment( $comment );
+		$respond_id = esc_attr( $args['respond_id'] );
+		$add_below  = esc_attr( $args['add_below'] );
+		/* This is to accommodate some themes that add an SVG to the Reply link like twenty-seventeen. */
+		$reply_text  = wp_kses(
+			$args['reply_text'],
+			array(
+				'svg' => array(
+					'class'           => true,
+					'aria-hidden'     => true,
+					'aria-labelledby' => true,
+					'role'            => true,
+					'xmlns'           => true,
+					'width'           => true,
+					'height'          => true,
+					'viewbox'         => true,
+				),
+				'use' => array(
+					'href'       => true,
+					'xlink:href' => true,
+				),
+			)
+		);
+		$before_link = wp_kses( $args['before'], wp_kses_allowed_html( 'post' ) );
+		$after_link  = wp_kses( $args['after'], wp_kses_allowed_html( 'post' ) );
+
+		$reply_url = esc_url( add_query_arg( 'replytocom', $comment->comment_ID . '#' . $respond_id ) );
+
+		$link = <<<HTML
+			$before_link
+			<a class="comment-reply-link" href="$reply_url" onclick="return addComment.moveForm( '$add_below-$comment->comment_ID', '$comment->comment_ID', '$respond_id', '$post->ID' )">$reply_text</a>
+			$after_link
+HTML;
+
+		return $link;
+	}
+
+	/**
+	 * Loop through all available fields and remove them.
+	 *
+	 * @param  array $fields - Default comment fields.
+	 * @return array $fields with no HTML.
+	 */
+	public function comment_form_default_fields( $fields ) {
+		foreach ( $fields as $field => $html ) {
+			remove_all_filters( "comment_form_field_{$field}" );
+			add_filter( "comment_form_field_{$field}", '__return_false', 100 );
+		}
+
+		return $fields;
+	}
+
+	/**
+	 * Clear FB comments on logout. wp-login.php doesn't clear these by default.
+	 *
+	 * @return void
+	 */
+	public function clear_fb_cookies() {
+		setcookie( 'wpc_fbc', ' ', time() - YEAR_IN_SECONDS, COOKIEPATH, COOKIE_DOMAIN, false, true );
```

Comment:
Shouldn't this be
```suggestion
		setcookie( 'wpc_fbc', ' ', time() - YEAR_IN_SECONDS, COOKIEPATH, COOKIE_DOMAIN, true, false );
```
<img width="1164" alt="Screenshot 2024-01-22 at 14 03 33" src="https://github.com/Automattic/jetpack/assets/7000684/570afed3-a947-4b2c-b6f1-ec5bef82fe74">

Reply:
The equivalent version of what we have on prod would be with `false`, `false` which are the defaults

Reply:
I updated the httponly value to true based on the php lint recommendations.
We should probably update both values to true. Are you aware of any issues that could possibly arise from these changes?